packages/create-project/tsconfig.json

```diff
@@ -4,7 +4,10 @@
     "outDir": "./lib",
     "rootDir": "./src",
     "resolveJsonModule": true,
-    "lib": ["dom"]
+    "lib": ["dom"],
+    "paths": {
+      "*": ["./src/types/*"]
```

Comment:
nit: we don't need the types here, the package would probably include types when it gets published.